trip-schema/src/main/java/global/maplink/trip/schema/v2/features/turnByTurn/TurnByTurnRequest.java

```diff
@@ -0,0 +1,18 @@
+package global.maplink.trip.schema.v2.features.turnByTurn;
+
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Data
+@Builder
+@RequiredArgsConstructor
+@NoArgsConstructor(force = true, access = PRIVATE)
+public class TurnByTurnRequest {
```

Comment:
Existe uma regra la no EBR onde o TurnByTurnRequest eh opcional, mas se for fornecido, o language tem que ser informado. O que acha de adicionar uma regra no validate do tripProblem pra esse cara?

Reply:
Poder ser, mas essa parte de validação fica tudo na V1 tem problema ?